xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -554,13 +576,13 @@ void CGUIFontTTF::DrawTextInternal(CGraphicContext& context,
         continue;
       }
 
-      if (alignment & XBFONT_TRUNCATED)
+      if (alignment & XBFONT_TRUNCATED || alignment & XBFONT_TRUNCATED_LEFT)
       {
```

Comment:
since the XBFONT_TRUNCATED and the XBFONT_TRUNCATED_LEFT case don't share anything, might be clearer to do
```
if ((alignment & XBFONT_TRUNCATED)
{ existing code }
else if (alignment & XBFONT_TRUNCATED_LEFT && itGlyph == glypghBegin)
{ new code - except the final }
```

Reply:
done